auth/store_test.go

```diff
@@ -621,6 +621,30 @@ func TestAuthDisable(t *testing.T) {
 	}
 }
 
+func TestAuthStatus(t *testing.T) {
```

Comment:
Could you rename this test to `TestIsAuthEnabled()`?

Reply:
I will work on these tonight.